_plugins/friendly_number.rb

```diff
@@ -0,0 +1,69 @@
+$unit = ['', 'K', 'M', 'B', 'T']
+
+module Jekyll
+  module FriendlyNumberFilter
+    # Formats a number with a unit 1750 => 1.7K
+    def friendly_number(number)
+      unless number.is_a? Numeric
+        return number
+      end
+
+      $iterations = 0
```

Comment:
In ruby, variables starting with `$` are global variables, so you've accidentally created a lot of global state here.

Reply:
derp, good to know.

Reply:
How should I handle the `$unit` variable?